Gemfile

```diff
@@ -13,7 +13,7 @@ gem "govuk_publishing_components", "~> 21.24.0"
 gem "rails", "~> 6.0.2"
 gem "slimmer", "~> 13.2.0"
 
-gem "sass-rails", "~> 6.0.0"
+gem "sass-rails", "< 6"
```

Comment:
Those size improvements look impressive! 👍 

Could you provide a bit more context about this change? I am not clear on why we'd prefer to use a version of sass-rails below 6 here. I wan't able to reproduce the change when pulling this branch, but I may have missed a step

Reply:
i've update the description. for reasons it doesn't fail locally, but see this [commit](https://github.com/alphagov/finder-frontend/pull/1953/commits/675f6efa1fc45ec96683d583025d7f55726219fb) for failed build

Reply:
pinned to specific version